.circleci/config.yml

```diff
@@ -108,47 +154,84 @@ commands:
 
 
 jobs:
-  release_staging:
+  test:
     docker:
       - image: circleci/node:11.15.0-stretch
     steps:
       - git_checkout_from_cache
-      - npm_install
+      - npm_install_from_cache
       - build
-      - docker
-      - k8s_deploy_staging
-    environment:
-      NODE_ENV: staging
-
-  release_production:
+  build_staging:
     docker:
       - image: circleci/node:11.15.0-stretch
     steps:
       - git_checkout_from_cache
-      - npm_install
+      - npm_install_from_cache
       - build
       - docker
+  build_production:
+    docker:
+      - image: circleci/node:11.15.0-stretch
+    steps:
+      - git_checkout_from_cache
+      - npm_install_from_cache
+      - build
+      - compress
+      - upload_artifact  # uploading the built code to s3 to create a backup of key services separate from Kubernetes deployment
+      - upload_checksum  # uploading compressed artifact checksum to cross match artifact fingerprint
+      - docker:
+          docker_image_tag: latest
+  deploy_staging:
+    docker:
+      - image: circleci/node:11.15.0-stretch
+    steps:
+      - k8s_deploy_staging      
+  deploy_production:
+    docker:
+      - image: circleci/node:11.15.0-stretch
+    steps:
       - k8s_deploy_production
-    environment:
-      NODE_ENV: staging
 
 workflows:
-  release_staging:
+  build_and_push_dev:
     jobs:
-      - release_staging:
-          context: binary-frontend-artifact-upload
+      - test:
+          filters:
+            branches:
+              only: /^dev/
+  build_deploy_staging:
+    jobs:
+      - build_staging:
+          context: deriv-app-artifact-upload
           filters:
             branches:
               ignore: /.*/
             tags:
               only: /^staging.*/
-  release_production:
+      - deploy_staging:
+          context: deriv-app-artifact-upload
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /^staging.*/
+          requires:
+            - build_staging
+  build_deploy_production:
     jobs:
-      - release_production:
-          context: binary-frontend-artifact-upload
+      - build_production:
+          context: deriv-app-artifact-upload
```

Comment:
@shudarshon-binary context needs to be binary-frontend-artifact-upload as the organisation is binary-com

Reply:
> @shudarshon-binary conflicts

yes, i have started to fix it